replica-lib/src/replica_confirmed_slots_server.rs

```diff
@@ -0,0 +1,121 @@
+use tonic;
+
+// tonic::include_proto!("accountsdb_repl");
```

Comment:
Do we need this?

Reply:
No -- I will remove the comment. The types are imported via accountsdb_repl_server